nipype/interfaces/afni/model.py

```diff
@@ -293,11 +293,14 @@ def _list_outputs(self):
             outputs['x1D'] = self._gen_fname(
                 suffix='.xmat.1D', **_gen_fname_opts)
 
+        if isdefined(self.inputs.cbucket):
+            outputs['cbucket'] = os.path.abspath(self.inputs.cbucket)
+
         outputs['reml_script'] = self._gen_fname(
             suffix='.REML_cmd', **_gen_fname_opts)
         # remove out_file from outputs if x1d_stop set to True
         if self.inputs.x1D_stop:
-            del outputs['out_file']
+            del outputs['out_file'], outputs['cbucket']
```

Comment:
if I may, `test_extra_Deconvolve.py` could be extended to check for the removal of `outputs['cbucket']`

Reply:
Thanks for reviewing this, @mvdoc ! It sounds like a new release is imminent, but if @effigies doesn't mind me using a circle cycle I'll submit a patch to extend the test.

Reply:
@emdupre nothing is building right now and won't probably merge anything more. feel free to  use circle, we can always cancel.

Reply:
Was going to say what @djarecka said. CI is not really the submitter's problem and we can cancel and restart as needed. I just try to give a heads up when doing that.

Reply:
Great, thanks for clarifying !